test/lit.cfg

```diff
@@ -1461,6 +1461,29 @@ if os.path.exists(libswiftCore_path):
     config.substitutions.append(('%target-static-stdlib-path', static_stdlib_path))
     lit_config.note('using static stdlib path: %s' % static_stdlib_path)
 
+# Set up testing with the standard libraries coming from the OS / just-built libraries
+# default Swift tests to use the just-built libraries
+# For remote runs, this is already supported in REMOTE_RUN_CHILD_DYLD_LIBRARY_PATH above
```

Comment:
I don't quite get this part. For remote runs, you'd have to *not* provide REMOTE_RUN_CHILD_DYLD_LIBRARY_PATH to get it to use the OS libraries, right?

Reply:
For remote runs, `REMOTE_RUN_CHILD_DYLD_LIBRARY_PATH` is all that matters - you can choose between the OS libraries and the just-built libraries by passing a different `remote_tmp_dir` parameter - you get to choose between the two options today. this new PR is for supporting said choice on a local target

Reply:
That's not correct. `remote_tmp_dir` is for uploads; you can't set it to /usr/lib/swift and have things work out. You'll need to honor this new option there too.

Reply:
ah, you're right, updated the PR accordingly.